### ISSUE TITLE
Support planned for DirectX raytracing?

### ISSUE THREAD

**Issue by oscarbg** [chaoticbob/tinyrenderers#12]

**Comment by L2NE-dev** [comment]
I planned to add support from Vulkan API side (unofficial).
About DXR, need deep consideration.

**Comment by oscarbg** [comment]
very nice news.. Sadly support for it could take a while ext. isn’t even posted, right?

**Comment by L2NE-dev** [comment]
@oscarbg don't forget about DXR fallback layers, they still updating this layer daily. DXR have two things: API itself and fallback layer for unsupported devices.

**Comment by oscarbg** [comment]
yes, I was talking about upcoming vulkan raytracing exntesions but I see vRt project now..
